pkg/backend/display/testdata/not-truncated/stack-output-resource-diff.json.interactive-100x80-cooked.stdout.txt

```diff
@@ -2,34 +2,34 @@
 
 
     <{%underline%}><{%fg 12%}>Type<{%reset%}>                 <{%underline%}><{%fg 12%}>Name<{%reset%}>           <{%underline%}><{%fg 12%}>Status<{%reset%}>     <{%underline%}><{%fg 12%}>Info<{%reset%}>[K
  <{%bold%}><{%reset%}>  <{%reset%}>  pulumi:pulumi:Stack  project-stack  <{%bold%}><{%reset%}><{%reset%}>           <{%reset%}>Configuration:<{%reset%}>[K
-
[1A[1A     <{%underline%}><{%fg 12%}>Type<{%reset%}>                 <{%underline%}><{%fg 12%}>Name<{%reset%}>                          <{%underline%}><{%fg 12%}>Status<{%reset%}>      <{%underline%}><{%fg 12%}>Info<{%reset%}>[K
```

Comment:
Why did all these change?

Reply:
The way the frame/render logic in the tree renderer worked was fragile (and broken if system messages needed to be displayed) when I used the same mechanism to display the download progress --- which is both more common than system messages, and happens at a different stage in the lifecycle of the engine --- I had to fix the way the redraw logic worked, which means different outputs in these files.

I will also investigate whether we can make these outputs not use escape codes, which will make comparing easier (unless you can read VT100/ANSI escape codes without a crib)

Reply:
I've made the change, LMK what you think, I can always revert it within the PR, it does feel more readable to me